perses/rjmc/geometry.py

```diff
@@ -586,8 +644,8 @@ def _torsion_and_angle_potential(self, xyz, atom, positions, involved_angles, in
             torsion_atom_position = xyz if torsion.atom4 == atom else positions[torsion.atom4.idx]
             internal_coordinates, _ = self._autograd_ctoi(atom_position, bond_atom_position, angle_atom_position, torsion_atom_position, calculate_jacobian=False)
             phi = internal_coordinates[2]*units.radians
```

Comment:
Hm.  Maybe we should *include* the impropers in the energy computation here in order to correctly capture ring closing and planarity.  We would still want to exclude them from picking which driver torsions should be used to place new atoms, however.